src/lib/glocal/detect-exec-layout.ts

```diff
@@ -137,29 +115,48 @@ export function detectExecLayout(input: Input): ExecScenario {
       toolProject: false,
       toolCurrentlyPresentInNodeModules: false,
       runningLocalTool: false,
-      thisProcessToolBin,
+      thisProcessScriptPath,
       project,
     }
   }
 
-  let projectToolBinRealPath = null
+  let projectToolPath: string | null = null
   try {
-    projectToolBinRealPath = fs.realpathSync(projectToolBinPath)
+    /**
+     * Find the project tool path by reverse engineering
+     * 1. find the tool package
+     * 2. find its local bin path
+     * 3. check that it AND the hoisted version at project dot-bin level exist
+     * 4. If yes yes yes then we've fonud our path!
+     *
+     * This logic is needed for Windows support because in Windows there are no
+     * symlinks we can follow for free.
+     */
+    const toolPackageJsonPath = requireResolveFrom(`${input.depName}/package.json`, projectDir)
+    const toolPackageDir = Path.dirname(toolPackageJsonPath)
+    const toolPackageRelativeBinPath: string | undefined = require(toolPackageJsonPath)?.bin[input.depName]
```

Comment:
Same concern as above about 1-to-1 bin/package-name naming

Reply:
Answered above